### PR TITLE
XW-4157 | Lack of autoplay video with adblock on

### PR DESCRIPTION
**Pull request by latata**
https://wikia-inc.atlassian.net/browse/XW-4157
@Wikia/x-wing

### CODE REVIEW

File: extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js [hunk 1]
```diff
@@ -10,6 +11,7 @@ require([
 	'wikia.articleVideo.featuredVideo.events',
 	require.optional('ext.wikia.adEngine.lookup.a9')
 ], function (
+	window,
```
Comment (kvas-damian) [inline extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js]: Please, don't use variable names that clash with global scope variables.

File: extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js [hunk 2]
```diff
@@ -95,4 +97,21 @@ require([
 	} else {
 		setupPlayer();
 	}
+
+	// XW-4157 PageFair causes pausing the video, as a workaround we play video again when it's paused
+	window.addEventListener('wikia.blocking', function () {
+		if(playerInstance) {
+			if(playerInstance.getState() === 'paused') {
```
Comment (kvas-damian) [inline extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js]: spaces after `if` ;)

File: extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js [hunk 3]
```diff
@@ -95,4 +97,21 @@ require([
...
+				playerInstance.play();
+			} else {
+				playerInstance.once('pause', function (event) {
+					// when video is paused because of PageFair pauseReason is undefined,
```
Comment (hakubo) [inline extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js]: thanks for this comment!

File: extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js (hunk 2 above)
Comment (hakubo) [reply extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js]: +1

**Review by hakubo — APPROVED** [review]

File: extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js [hunk 4]
```diff
@@ -95,4 +97,21 @@ require([
 	} else {
 		setupPlayer();
 	}
+
+	// XW-4157 PageFair causes pausing the video, as a workaround we play video again when it's paused
+	window.addEventListener('wikia.blocking', function () {
+		if(playerInstance) {
+			if(playerInstance.getState() === 'paused') {
+				playerInstance.play();
```
Comment (kvas-damian) [inline extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js]: Please make sure non-autoplay still works after these changes.

Comment (latata) [reply extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js]: it still works fine

**Review by kvas-damian — APPROVED** [review]

**Pull request merged**